_posts/2018-03-02-this-week-in-rustfest-1-anonymouse-abstract-first-batch-locals.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: "This week in RustFest 1: First batch, local communities and CfP process explained"
+authors:
+  - dns2utf8
+  - imperio
+  - pnkfelix
+---
+
+With some further adoo, this weeks news.
```

Comment:
"adoo"?

Reply:
like the opposite of "without further ado"